### PR TITLE
Removed bower

### LINKED ISSUE #3957: Remove bower
Bower is deprecated and holding us back from using embroider. Use yarn to pull in the packages instead of bower

Parent: #3837 

https://bower.io/blog/2017/how-to-migrate-away-from-bower/

### PR DESCRIPTION
**Pull request by gittysachin**
Fixes #3957 

#### Short description of what this resolves:
As bower is deprecated and not letting us use embroider, replacing ```bower_component``` with ```node_modules@bower_components``` will allow us to use those same packages after installing using yarn.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Remove post install hook

**Comment by iamareebjamal** [comment]
Many of these dependencies can be installed using yarn. So, remove them as bower components. Use sentry SDK instead of raven JS

**Comment by kushthedude** [comment]
Install croppie using yarn

File: app/adapters/application.js [hunk 1]
```diff
@@ -20,7 +19,7 @@ export const fixFilterQuery = query  => {
   return query;
 };
 
-export default JSONAPIAdapter.extend(HasManyQueryAdapterMixin, AdapterFetch, FastbootAdapter, {
+export default JSONAPIAdapter.extend(HasManyQueryAdapterMixin, FastbootAdapter, {
```
Comment (kushthedude) [inline app/adapters/application.js]: Unrelated changes are appearing in your PR, Please rebase correctly.

File: package.json [hunk 2]
```diff
@@ -15,8 +15,7 @@
     "test": "ember test",
     "exam": "ember exam --launch=chrome --random",
     "lint:js": "eslint --fix app ember-cli-build.js testem.js tests",
-    "lint:scss": "sass-lint -c .sass-lint.yml --verbose",
-    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'bower_components', 'junction') } catch (e) { }\""
+    "lint:scss": "sass-lint -c .sass-lint.yml --verbose"
```
Comment (kushthedude) [inline package.json]: Where are changes related to bower components in here ?

File: app/adapters/application.js (hunk 1 above)
Comment (gittysachin) [reply app/adapters/application.js]: @kushthedude, can you please guide me on gitter. I'm having some problems related to rebase. That would be so helpful.

**Comment by iamareebjamal** [comment]
status?

**Comment by kushthedude** [comment]
Stale !
